app/views/articles/_about_author.html.erb

```diff
@@ -1,44 +1,48 @@
 <div>
-  <p class="mb-4 color-base-60 fs-s">Posted on <%= published %> by:</p>
+  <p class="mb-4 color-base-60 fs-s">
+    Posted on <time datetime="<%= @article.published_timestamp %>"><%= @article.readable_publish_date %></time> by:
+  </p>
 
-  <div class="flex">
-    <a href="/<%= @user.username %>" class="crayons-avatar crayons-avatar--2xl shrink-0 mr-4">
-      <img class="crayons-avatar__image" src="<%= ProfileImage.new(@user).get(width: 180) %>" alt="<%= @user.username %> profile" />
-    </a>
-    <div>
-      <h3 class="fs-l s:fs-xl m:fs-2xl l:fs-3xl fw-medium"><a href="/<%= @user.username %>" class="crayons-link"><%= @user.name %></a></h3>
-      <p class="fw-medium mb-2">
-        <a href="/<%= @user.username %>" class="crayons-link crayons-link--secondary">
-          @<%= @user.username %>
-        </a>
-      </p>
-      <% if @user.summary %>
-        <p class="color-base-70 mb-4"><%= @user.summary %></p>
-      <% end %>
+  <% cache("article-about-author-#{@user.id}-#{@user.updated_at}", expires_in: 100.hours) do %>
+    <div class="flex">
+      <a href="/<%= @user.username %>" class="crayons-avatar crayons-avatar--2xl shrink-0 mr-4">
```

Comment:
**nitpick (please change):** Instead of manually constructing URL (even if they are simple), please use our custom `URL` helper module. 

```suggestion
      <a href="<%= URL.user(@user) %>" class="crayons-avatar crayons-avatar--2xl shrink-0 mr-4">
```